starter-core/src/test/groovy/io/micronaut/starter/feature/FeatureSpec.groovy

```diff
@@ -41,7 +41,8 @@ class FeatureSpec extends BeanContextSpec {
         JdkVersion javaVersion = javaVersionForFeature(feature.getName())
         Language language = Language.JAVA
         if (feature instanceof LanguageSpecificFeature) {
-            language = ((LanguageSpecificFeature) feature).getRequiredLanguage();
+            // TODO: support more languages
+            language = ((LanguageSpecificFeature) feature).getRequiredLanguages()[0];
```

Comment:
I don't understand this change. Why do we need an array of languages?

Reply:
not there anytmore